sale_payment_sheet/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "name": "Sale payment sheet",
     "summary": "Allow to create invoice payments to commercial users without "
     "accounting permissions",
-    "version": "15.0.1.1.2",
+    "version": "16.0.1.1.2",
```

Comment:
Hi, you should restart the version from 16.0.1.0.0
```suggestion
    "version": "16.0.1.0.0",
```

Reply:
ok fixed